test/test_utils/security.py

```diff
@@ -548,6 +801,35 @@ def run_scan(ecr_client, image):
         raise TimeoutError(f"ECR Scan is still in {scan_status} state. Exiting.")
 
 
+def wait_for_enhanced_scans_to_complete(ecr_client, image):
```

Comment:
If there is more time to implement this, is there a boto3 Waiter that can be used to wait for scan completion instead of a while-loop?

The original basic ECR scan was implemented with a while loop because we didn't know better.

Reply:
Can we post pone the waiter implementation for future?